js/canvas.js

```diff
@@ -0,0 +1,367 @@
+import { arrayOfRectangles } from './rectangles.js';
+import { canvasWindowIndent, startedRectangleFillColor, redColorInHex, stickyDistance } from './shared/constants.js';
+
+export class Canvas {
+  constructor(selectedCanvas) {
+    this.canvas = selectedCanvas;
+    this.ctx = this.canvas.getContext('2d');
+    this.draggingRect;
+    this.isDragOk = false;
+    this.offsetX;
+    this.offsetY;
+  }
+
+  start() {
+    this.setSize();
+    this.setOffsets_X_and_Y();
+    this.drawRectangles();
+    this.canvas.addEventListener('mousedown', this.mouseDownHandler.bind(this));
+    this.canvas.addEventListener('mousemove', this.mouseMoveHandler.bind(this));
+    this.canvas.addEventListener('mouseup', this.mouseUpHandler.bind(this));
+  }
+
+  setSize() {
+    const { innerWidth: width, innerHeight: height } = window;
+
+    this.canvas.width = width - canvasWindowIndent;
+    this.canvas.height = height - canvasWindowIndent;
+  }
+
+  setOffsets_X_and_Y() {
```

Comment:
what style of naming do we take as a basis? camelCase or under_score?

Reply:
camelCase, setOffsetsXandY or setOffsetsXAndY need to write?